	- do not use /lib for kernel source dir detection
	- drop some non-compiling card drivers
	- do not run depmod and stuff
--- mISDN-1_1_9_2.orig/Makefile	2009-06-09 10:41:07.000000000 +0200
+++ mISDN-1_1_9_2/Makefile	2011-07-11 15:04:00.996001762 +0200
@@ -14,9 +14,14 @@ ifndef KVERS
 KVERS:=$(shell uname -r)
 endif
 
+ifndef LINUX
 MODS=/lib/modules/$(KVERS)
 LINUX=$(MODS)/build
 LINUX_SOURCE=$(MODS)/source
+else
+LINUX_SOURCE=${LINUX}
+endif
+
 UPDATE_MODULES=$(shell which update-modules)
 MODULES_UPDATE=$(shell which modules-update)
 DEPMOD=$(shell which depmod)
@@ -30,15 +35,15 @@ MISDN_SRC=$(MISDNDIR)/drivers/isdn/hardw
 
 CONFIGS+=CONFIG_MISDN_DRV=m 
 CONFIGS+=CONFIG_MISDN_DSP=m 
-CONFIGS+=CONFIG_MISDN_HFCMULTI=m 
-CONFIGS+=CONFIG_MISDN_HFCPCI=m
+#CONFIGS+=CONFIG_MISDN_HFCMULTI=m 
+#CONFIGS+=CONFIG_MISDN_HFCPCI=m
 CONFIGS+=CONFIG_MISDN_HFCUSB=m
 CONFIGS+=CONFIG_MISDN_XHFC=m
-CONFIGS+=CONFIG_MISDN_HFCMINI=m
-CONFIGS+=CONFIG_MISDN_W6692=m
-CONFIGS+=CONFIG_MISDN_SPEEDFAX=m
-CONFIGS+=CONFIG_MISDN_AVM_FRITZ=m
-CONFIGS+=CONFIG_MISDN_NETJET=m
+#CONFIGS+=CONFIG_MISDN_HFCMINI=m
+#CONFIGS+=CONFIG_MISDN_W6692=m
+#CONFIGS+=CONFIG_MISDN_SPEEDFAX=m
+#CONFIGS+=CONFIG_MISDN_AVM_FRITZ=m
+#CONFIGS+=CONFIG_MISDN_NETJET=m
 CONFIGS+=CONFIG_MISDN_DEBUGTOOL=m 
 
 #CONFIGS+=CONFIG_MISDN_NETDEV=y
@@ -52,9 +57,6 @@ all: VERSION test_old_misdn test_new_mis
 	export MINCLUDES=$(MISDNDIR)/include ; export MISDNVERSION=$(MISDNVERSION); make -C $(LINUX) SUBDIRS=$(MISDN_SRC) modules $(CONFIGS)  
 
 install: all modules-install misdn-init
-	$(DEPMOD) 
-	$(UPDATE_MODULES)
-	$(MODULES_UPDATE)
 	make -C config install
 
 modules-install:
